### PR TITLE
newrelic npi role

### PR DESCRIPTION
**Pull request by fredsmith**

### CODE REVIEW

File: playbooks/roles/newrelic-npi/meta/main.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+#
+##
+# Role includes for role newrelic-npi
+# 
+# Example:
+#
+ dependencies:
+   - {
+   role: newrelic 
+ }
```
Comment (feanil) [inline playbooks/roles/newrelic-npi/meta/main.yml]: You don't need the braces.

File: playbooks/roles/newrelic-npi/defaults/main.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+# 
+
+#
+# vars are namespace with the module name.
+#
+newrelic-npi_role_name: newrelic-npi
+
+#
+# OS packages
+newrelic-npi_debian_pkgs: []
```
Comment (feanil) [inline playbooks/roles/newrelic-npi/defaults/main.yml:21]: Do we need to ensure the `node` package is installed as a part of this role?

Comment (fredsmith) [inline playbooks/roles/newrelic-npi/defaults/main.yml:21]: the installer actually installs a local copy of node!

Comment (feanil) [inline playbooks/roles/newrelic-npi/defaults/main.yml:21]: :hankey:

**Comment by feanil** [comment]
:+1:

**Comment by feanil** [comment]
:sheep: :it:

**Pull request merged**